components/single_image_view.jsx

```diff
@@ -170,10 +168,22 @@ export default class SingleImageView extends React.PureComponent {
 
         let viewImageModal;
         let loadingImagePreview;
-
         let fadeInClass = '';
-        let imageStyle = {height: previewHeight / 100};
-        let imageContainerStyle = {height: previewHeight / 100};
+
+        let height = previewHeight;
+        if (height < PREVIEW_IMAGE_MIN_DIMENSION) {
+            height = PREVIEW_IMAGE_MIN_DIMENSION;
+        }
+
+        let width = previewWidth;
+        if (width < PREVIEW_IMAGE_MIN_DIMENSION) {
+            width = PREVIEW_IMAGE_MIN_DIMENSION;
+        }
+
+        let imageStyle = {height};
+        const imageContainerStyle = {width};
```

Comment:
Set static width to `image-container` to properly scale the image based on height (or max-height). Currently, it's not scaling (whenever the viewport width is greater than the image width) but only maintained the ratio and then just hide the lower portion of the image.